craft_parts/infos.py

```diff
@@ -100,6 +102,11 @@ def target_arch(self) -> str:
         """Return the architecture used for debs, snaps and charms."""
         return self._machine["deb"]
 
+    @property
+    def base(self) -> str:
```

Comment:
Is this just required for repo?

Reply:
Yes, it's only used in `fetch_stage_packages` to filter package names for that base.